published/007-2016-March-13.md

```diff
@@ -0,0 +1,58 @@
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [second week of March](//github.com/ipfs/pm/issues/97):
+
+## Updates
+
+### [__Repo__](//github.com/)
+
+### Orbit
+
+[Orbit](https://github.com/haadcode/orbit) is back in active development and going through a major code base overhaul. It now uses orbit-db as it's database layer.
+
+[orbit-db](https://github.com/haadcode/orbit-db) is a KV-store and Event Log on top of IPFS which allows developers to use IPFS as a database. Last week orbit-db got big performance and stability improvements and is now using CRDTs for eventual consistency.
+
+### [js-multiaddr](https://github.com/jbenet/js-multiaddr)
+
+_@diasdavid:_ I don't know what these mean. Why are these changes important? Can you explain them in English a bit more? Also, these weren't mentioned in the sprint hangout as things done last week - why not?
+
+The [js-multiaddr](https://github.com/jbenet/js-multiaddr) implementation was upgraded, now that it now matches go-ipfs features. This interoperability  which means protocols whose code's required a varint 
+
+mafmt now does uTP and WebSockets validation as well
+
+new libp2p-swarm released, check it out :)
+
+
+### Go-IPFS
+
+A nasty bug in our networking code was fixed this week. An issue in yamux (our primary stream multiplexer) would cause code to hang when opening a new stream if there were too many in flight stream opens. As a result of this, large file transfers (ipfs refs -r, ipfs get, and so on) would hang.
+
+IPFS 0.4.0 is now very close to shipping: we ran @dignifiedquire's randor tool quite extensively and are more confident in the repo operations that have changed since 0.3.11. The release now has a somewhat short checklist of things that are blocking the official release.
+
+As a small side project, @whyrusleeping started benchmarking each of our possible datastore implementations with ds-bench. The results will help us improve the performance of our storage moving forward. On that same topic, @whyrusleeping started an experiment in new and exciting datastores and wrote a datastore implementation that can be backed by a sql database. He then spun up a postgresql database (from docker) and ran the benchmarks against it, without any sort of tuning the initial performance metrics showed that it was around three time as fast as our current on-disk datastore. (Note: the metrics are simply testing the performance of writing random 256k binary blobs to each datastore).
```

Comment:
benchmark code is here: https://github.com/whyrusleeping/ds-bench

After 0.4.0 i'm going to work more on that perf